web-ui/src/components/menu/Menu.jsx

```diff
@@ -334,151 +303,157 @@ function Menu() {
-          {createListJsx(feedbackLinks, true)}
-        </Collapse>
-        {hasReportPermission && (
-          <React.Fragment>
-            <ListItem
-              button
-              onClick={toggleReports}
-              className={classes.listItem}
-            >
-              <ListItemText primary="REPORTS" />
-            </ListItem>
-            <Collapse in={reportsOpen} timeout="auto" unmountOnExit>
-              {createListJsx(getReportLinks(), true)}
-            </Collapse>
-          </React.Fragment>
-        )}
+        <div>
+          {createLinkJsx('/', 'HOME', false)}
+          {isAdmin && (
+            <>
+              <ListItem onClick={toggleAdmin} className={classes.listItem}>
+                <ListItemText primary="ADMIN" />
+              </ListItem>
+              <Collapse in={adminOpen} timeout="auto" unmountOnExit>
+                {createListJsx(adminLinks, true)}
+                {isAdmin && (
+                  <ListItem
+                    className={classes.listItem}
+                    onClick={openHoursUpload}
+                    style={{ marginLeft: '1rem' }}
+                  >
+                    Upload Hours
+                  </ListItem>
+                )}
+              </Collapse>
+            </>
+          )}
+          {createLinkJsx('/checkins', 'CHECK-INS', false)}
+          <ListItem onClick={toggleDirectory} className={classes.listItem}>
+            <ListItemText primary="DIRECTORY" />
+          </ListItem>
+          <Collapse in={directoryOpen} timeout="auto" unmountOnExit>
+            {createListJsx(directoryLinks, true)}
+          </Collapse>
+          <ListItem onClick={toggleFeedback} className={classes.listItem}>
+            <ListItemText primary="FEEDBACK" />
+          </ListItem>
+          <Collapse in={feedbackOpen} timeout="auto" unmountOnExit>
+            {createListJsx(feedbackLinks, true)}
+          </Collapse>
+          {hasReportPermission && (
+            <React.Fragment>
+              <ListItem
+                button
+                onClick={toggleReports}
+                className={classes.listItem}
+              >
+                <ListItemText primary="REPORTS" />
+              </ListItem>
+              <Collapse in={reportsOpen} timeout="auto" unmountOnExit>
+                {createListJsx(getReportLinks(), true)}
+              </Collapse>
+            </React.Fragment>
+          )}
+        </div>
+        <SchemeToggle />
       </List>
     </div>
   );
 
   return (
-    <Root className={classes.root}>
-      <CssBaseline />
-      <AppBar position="fixed" className={classes.appBar}>
-        <Toolbar>
-          <IconButton
-            color="inherit"
-            aria-label="open drawer"
-            edge="start"
-            onClick={handleDrawerToggle}
-            className={classes.menuButton}
-            size="large"
+    <div className={classes.root}>
+      <CssVarsProvider>
```

Comment:
In `App.jsx` [we use this](https://github.com/objectcomputing/check-ins/pull/2335/files#diff-a11cac4bbb0df4570e6a2061d4e78c126ec3bd5e2af9e1533eae376032386e26R78) with `theme`. Is that why we don't need to supply the `theme` here as well, since we already have it from higher up in the component tree?

Reply:
Yes, but this was the fix that I found to include the ThemeToggle in Menu, however I could have possibly made that a `props.children` perhaps, I'm so used to `<slot />` I often forget about the children.

Reply:
refactored this to remove the need for `CssVarsProvider` in Menu.jsx